### ISSUE TITLE
output*mask in warp

### ISSUE THREAD

**Issue by cxy7tv** [NVlabs/PWC-Net#99]
Hi, I have no idea why mask is needed following code in PWC-Net/PyTorch/models/PWCNet.py
 mask = torch.autograd.Variable(torch.ones(x.size())).cuda()
        mask = nn.functional.grid_sample(mask, vgrid)

        # if W==128:
            # np.save('mask.npy', mask.cpu().data.numpy())
            # np.save('warp.npy', output.cpu().data.numpy())
        
        mask[mask<0.9999] = 0
        mask[mask>0] = 1
        
        return output*mask

**Comment by poppinjie** [comment]
It seems that getting rid of it has the same effect.
